source/manual/alerts/fastly-error-rate.html.md

```diff
@@ -27,6 +27,10 @@ investigation is to examine the Fastly CDN logs.
 - `ssh logs-cdn-1.management.production`
 - `cd /mnt/logs_cdn` to access log files
 
+Alternatively you can look in Kibana with the query:
+
+`@fields.application:"govuk-cdn-logs-monitor"`
```

Comment:
you could add a link to a dashboard if you wanted to, such as https://kibana.publishing.service.gov.uk/kibana#/dashboard/elasticsearch/GOV.UK%20CDN%20logs, otherwise it looks good to me